application/controllers/bids.py

```diff
@@ -83,6 +84,8 @@ def post(self):
             abort(404, description='That item or user does not exist')
         else:
             bid = Bid(**args)
+            dt = trunc(datetime.now().timestamp())
```

Comment:
curious about this `trunc` function . . . `trancate`?

Reply:
This function removes any trailing decimals as a timestamp will give you seconds and milliseconds. We only want to return the seconds so I went ahead and chopped off the milliseconds.